lib/src/components/NxTab/NxTab.scss

```diff
@@ -6,16 +6,54 @@
  */
  @import '../../scss-shared/nx-colors';
  @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
 
 .nx-tab {
-  border-bottom: 4px solid transparent;
+  @include nx-truncate-ellipsis;
+
+  box-sizing: border-box;
+  border-bottom: 2px solid transparent;
   cursor: pointer;
   display: inline-block;
+  flex-shrink: 1;
   list-style: none;
-  padding: $nx-spacing-xs $nx-spacing-l;
+  margin: 0 $nx-spacing-md -1px 0;
+  // subtract border thickness for consistent spacing
+  padding: $nx-spacing-md $nx-spacing-l ($nx-spacing-md - 2px) $nx-spacing-l;
+  text-align: center;
+  min-width: 80px;
+  max-width: 240px;
 
   &.active {
-    border-bottom-color: $nx-background-color-interactive;
-    font-weight: bold;
+    @include semi-bold();
+
+    border-bottom-color: #045dd1;
+    color: #045dd1;
+  }
+
+  &:hover {
+    @include semi-bold();
+  }
+
+  &:focus {
+    border-bottom-width: 1px;
+    box-shadow: 0 0 3px 1px transparentize(#02a6d9, 0.7);
+    margin-bottom: 0;
+    outline: solid 1px #02a6d9;
+  }
+
+  a:hover, a:focus {
+    @include bold();
+
+    letter-spacing: 0
+  }
```

Comment:
What is this block? We aren't using anchor tags here, and I don't see why `letter-spacing` would need to be customized.

Reply:
> What is this block? We aren't using anchor tags here, and I don't see why `letter-spacing` would need to be customized.

I was part of a failed attempt to solve the text shift issue, it should have been removed: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/86df685593587a13a03259835ba70ed4bdd6b29b

Reply:
> I was part of a failed attempt to solve the text shift issue, it should have been removed: 86df685

Shouldn't the whole block be removed?

Reply:
> > I was part of a failed attempt to solve the text shift issue, it should have been removed: [86df685](https://github.com/sonatype/sonatype-react-shared-components/commit/86df685593587a13a03259835ba70ed4bdd6b29b)
> 
> Shouldn't the whole block be removed?

Oh yeah: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/be57bcf2d4e48df8773383a536152955cb3a6072